jellyfin-helpers/jellyfin_helpers/jellyfin_api.py

```diff
@@ -59,6 +62,13 @@ def _prepare_request_url(
             kwargs.update(parameters)
         return _build_url(server_url=self.server_url, path=path, kwargs=kwargs)
 
+    def _send_delete_request(
+        self, path: str, parameters: Optional[Dict] = None
+    ) -> None:
+        url = self._prepare_request_url(path=path, parameters=parameters)
+        response = requests.delete(url, headers={"X-Emby-Token": self.token})
+        response.raise_for_status()
```

Comment:
@AkagiShigeru It's giving a 204 like the API documentation says. However, when I look at the playlist (or re-run the code), it still detects the item. Any ideas? Can you check our version vs newest to see if it's a resolved API issue?

https://api.jellyfin.org/#tag/Playlists/operation/RemoveItemFromPlaylist

Reply:
This seems to be a [known issue](https://github.com/jellyfin/jellyfin/issues/2130) that is likely still present in our current server version (10.8.13.0). Even debug logging won't help to resolve this. 
I created a task to update the Jellyfin server instance to a very recent version to see if the issue is resolved.